### ISSUE TITLE
Accessibility: Revision selector needs a label

### ISSUE THREAD

**Issue by belcherj** [Automattic/simplenote-electron#1396]
The revision selector input in this [component](https://github.com/Automattic/simplenote-electron/blob/develop/lib/revision-selector/index.jsx) does not have a label.  It needs a label that is hidden

```<input class="slider" type="range" min="0" max="5" value="5">```

### What
Add a hidden lablel to the revision selector input

### Why
To make it accessible to screen readers

### How
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/forms/Basic_form_hints

<img width="454" alt="Screen Shot 2019-06-21 at 1 54 44 PM" src="https://user-images.githubusercontent.com/6817400/59941953-81489400-942c-11e9-853a-abf78798ce6f.png">

**Comment by alemagio** [comment]
Hi, i would like to work on this issue.

**Comment by belcherj** [comment]
PR's are welcome.  Sorry I didn't respond sooner!

**Comment by codebykat** [comment]
See also #980

**Comment by alemagio** [comment]
I'm sorry i could not work on this but i had some trouble.
If anyone else want to it is ok for me

**Comment by tbourrely** [comment]
Hi, I would be happy to work on this if possible :)

**Comment by dmsnell** [comment]
@tbourrely go for it! what do you need from us to get started?

you are welcome to fork the repo and submit a PR 😄

**Comment by tbourrely** [comment]
Well, once I can get my environment working, I will be good to go 😄  !

**Comment by sandymcfadden** [comment]
Looks like things have been refactored since this was opened. This input is now in the [Slider component](https://github.com/Automattic/simplenote-electron/blob/develop/lib/components/slider/index.tsx).

May want to approach it similar to how we work with the [Search input](https://github.com/Automattic/simplenote-electron/blob/develop/lib/search-field/index.tsx#L81).
